R/table-subscript.R

```diff
@@ -773,6 +773,10 @@ updateSpanIfNecessary <- function(y, x.attributes, evaluated.args) {
         if (length(evaluated.args) > 2L) evaluated.args <- evaluated.args[1:2]
         span.df <- mapply(subscriptSpanDF, span.attribute, evaluated.args, SIMPLIFY = FALSE)
         span.df <- Filter(ncol, span.df)
+        if (is.null(ncol(y)) && !is.null(span.df$columns)) {
+            span.df$rows <- span.df$columns
+            span.df$columns <- NULL
+        }
```

Comment:
This is in the table-subscript file and not the transpose file. However, it looks like a transpose operation (replace row with column information)? Is this correct? If so, it could do with a comment indicating why it is needed here?

Reply:
Yep, this doesn't go in table-transpose because `t()` wasn't called - its a result of `drop`. I've added a comment in [95fc227](https://github.com/Displayr/verbs/pull/120/commits/95fc2275f491e8a6accd5e28cea508126af19b73)